packages/beacon-node/src/chain/blocks/index.ts

```diff
@@ -110,7 +110,7 @@ export async function processBlocks(
     if (!(err instanceof BlockError)) {
       this.logger.error("Non BlockError received", {}, err);
     } else if (!opts.disableOnBlockError) {
-      this.logger.error("Block error", {slot: err.signedBlock.message.slot}, err);
+      this.logger.warn("Block error", {slot: err.signedBlock.message.slot}, err);
```

Comment:
Reference: https://github.com/ChainSafe/lodestar/issues/3062
Block errors are sometimes expected, especially for unstable/testnet networks and `error` doesn't feel like the correct verbosity as the user cannot be expected to do something about it. As mentioned in the issue, `warn` or `info` may be more appropriate. Should we also consider removing the stacktrace?

Reply:
This log here does not make much sense to be above debug level IF ONLY the consumers handle the error properly. Invalid blocks can be submitted via:
- gossip: a peer is free to submit and invalid block. That's an uncontrolled event the user can't do much about, so we may want to not log it at all. However, if there's a consensus bug the node will just freeze without warning
- sync: same as above
- API: this _should_ never happen as long as the API is trusted. The API also logs errors so this will probably be a duplicate

For now, let's manually review every caller of this path and check if they already log the error and at what level

Reply:
This log has been moved to debug
- https://github.com/ChainSafe/lodestar/pull/5485